arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -347,7 +345,10 @@ public List<Module> getModulesToTrace() {
                 stp,
                 trm,
                 txnData,
-                wcp))
+                wcp
+                // l2Block
```

Comment:
L2block is a count onlyoduke, shouldn't be traced

Reply:
I added it in comments to have the same list as in `modulesToCount` (same length.) We can remove it.